modules/phase_field/doc/content/source/meshgenerators/SphereSurfaceMeshGenerator.md

```diff
@@ -0,0 +1,22 @@
+# SphereSurfaceMeshGenerator
+
+!syntax description /Mesh/SphereSurfaceMeshGenerator
+
+This object generates a 2D mesh approximating the surface of a sphere in 3D space
+using TRI3 triangle elements. Both center and radius of the sphere may be specified.
+
+!media phase_field/sphere.gif style=width:30%;margin-left:20px;float:right;
+       caption=Targeted mesh refinement prior to nucleus insertion.
+
+The mesh is constructed by iterative definement of an initial icosahedron (depth 0)
```

Comment:
```suggestion
The mesh is constructed by iterative refinement of an initial icosahedron (depth 0)
```

Reply:
Wait - are you saying "definement" is actually what you want?

Reply:
Uh.. no. Did I mess up accepting the suggestion? It is refinement for sure. Thanks for spotting that typo.

Reply:
Ok, "refinement" is what ended up in ac746d1. I combined the accepted suggestion commit and my fixes and tagged you as a co-author of that commit.